packages/0x.js/src/order_watcher/remaining_fillable_calculator.ts

```diff
@@ -0,0 +1,71 @@
+import {
+    SignedOrder,
+} from '../types';
+import { BigNumber } from 'bignumber.js';
+export class RemainingFillableCalculator {
+    private _signedOrder: SignedOrder;
+    private _isMakerTokenZRX: boolean;
```

Comment:
I'd rather pass a ZRX address here and check inside

Reply:
Yeah I had this originally, it felt a bit clunky as it checked and stored isMakerTokenZRX internally anyway. From an external developer using the calculator it reminds the caller there is complexity when ZRX is Maker Token. Providing just zrxAdress loses this reminder and felt awkward.